packages/widgetbook/lib/src/widgets/expanders/expander_row.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook/src/models/organizers/organizers.dart';
+
+import 'package:widgetbook/src/widgets/expanders/expand_button.dart';
+
+class ExpanderRow extends StatelessWidget {
+  const ExpanderRow({
+    Key? key,
+    this.size,
+    required this.organizers,
+  }) : super(key: key);
```

Comment:
here it would be great to have this as a private constructor and define two factory constructors to define a `ExpanderRow.small` and `ExpanderRow.large` which uses the private constructor with specific `size` values (e.g. 18 for the `small` and 32 for the `large` `ExpanderRow`)

Reply:
Ah that's cool, that way the sizes are enforced